meos/src/general/span.c

```diff
@@ -1972,7 +1972,7 @@ span_ne(const Span *s1, const Span *s2)
  * @ingroup meos_internal_setspan_comp
  * @brief Return -1, 0, or 1 depending on whether the first span is less than,
  * equal, or greater than the second one
- * @param[in] s1,s2 Sets
+ * @param[in] s1,s2 Spans
```

Comment:
I think this might still cause conflicts. Either you change this here, or I accept this PR and you can change this in the other one. The other one still needs a possible commit for some `free` calls

Reply:
This function is for the qsort algorithm and thus it is NOT supposed to be used as an external function.